src/main/core-options.js

```diff
@@ -237,7 +237,7 @@ const options = {
     category: CATEGORY_GLOBAL,
     default: 2,
     description: "Number of spaces per indentation level.",
-    range: { start: 0, end: Infinity, step: 1 },
+    range: { start: 1, end: Infinity, step: 1 },
```

Comment:
We can't validate this option?

Reply:
I see some `validate` calls in `options-normalizer.js`, but looks like they don't check this `range` property.